modules/server/src/network/index.ts

```diff
@@ -1,46 +1,61 @@
 import { makeExecutableSchema } from '@graphql-tools/schema';
-import { SupportedAggregation, SUPPORTED_AGGREGATIONS_LIST } from './common';
 import { createResolvers } from './resolvers';
-import { getAllFieldTypes } from './setup/fields';
+import { SUPPORTED_AGGREGATIONS } from './setup/constants';
+import { normalizeFieldTypes } from './setup/fields';
 import { fetchAllNodeAggregations } from './setup/query';
 import { createTypeDefs } from './typeDefs';
 import { NetworkConfig } from './types/setup';
 
-export let ALL_NETWORK_AGGREGATION_TYPES_MAP: Map<string, SupportedAggregation> = new Map();
+/**
+ * Map of all available fields with associated aggregation type
+ */
+export let ALL_NETWORK_AGGREGATION_TYPES_MAP: Map<string, keyof typeof SUPPORTED_AGGREGATIONS> =
```

Comment:
`let` seems atypical here. if I didn't know better I'd ask if an AI assistant suggested this here